### DISCUSSION TITLE
Migration from IE Web Browser control to WebView2

### DISCUSSION THREAD

**Discussion by ukalhapu** [MicrosoftEdge/WebView2Feedback#3644] [General]
We have legacy VB.NET application using IE WebBrowser control. 
We have extended/customized/wrapped the existing WebBrowser control for application specific needs.
Now we want to migrate to WebView2.
Is there standard migration guide/white paper published by Microsoft to ease migration from Web Browser control to WebView2 ?

**Comment by aboma** [comment]
I have done the same thing - IE to Webview2 migration. If you have any questions let me know.

**Comment by ukalhapu** [comment]
Hi Adrian, 
Thanks for the response. 
Could you please help with the issue [#3625](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3625)

**Comment by novac42** [comment]
Hi @ukalhapu right now we don't have official migration guide/doc published on this topic. I will move this post to discussion to see if there're any other developers who want to share their migration learnings. Also @aboma @ukalhapu would you like to share which APIs you primarily use in IE Web Browser control?
